DBAccess/SqliteAccess.cs

```diff
@@ -1,36 +1,98 @@
 ﻿using System;
 using System.Collections.Generic;
 using System.Data;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
+using System.Data.SQLite;
+using System.Globalization;
 
 namespace DBAccess
 {
     class SqliteAccess : DBAccess
     {
+        private SQLiteConnection connection;
         public SqliteAccess(string connectionString) : base(connectionString)
         {
+            SQLiteConnectionStringBuilder connectionstring = new SQLiteConnectionStringBuilder(connectionString);
+            try
+            {
+                this.connection = new SQLiteConnection(connectionstring.ConnectionString);
+            }
+            catch (SQLiteException e)
+            {
+                this.ProcessException(e);
+            }
+            this.Connect();
+
 
         }
         public override void Connect()
         {
-            throw new NotImplementedException();
+            if (this.connection.State == ConnectionState.Open)
+                return;
+            try
+            {
+                connection.Open();
+            }
+            catch (SQLiteException e)
+            {
+                this.ProcessException(e);
+            }
         }
 
         public override void Disconnect()
         {
-            throw new NotImplementedException();
+            try
+            {
+                connection.Close();
+            }
+            catch (SQLiteException e)
+            {
+                this.ProcessException(e);
+            }
         }
 
-        public override DataTable SqlQuery(string sql, IDictionary<string, object> parameters)
+        public override DataTable SqlQuery(string sql, IDictionary<string, Object> parameters)
         {
-            throw new NotImplementedException();
+            this.CleanStatus();
+            SQLiteCommand cmd = this.AddParameters(sql, parameters);
+
+            SQLiteDataAdapter oDataAdapter = new SQLiteDataAdapter(sql, this.connection);
+            DataTable result = new DataTable();
+            result.Locale = CultureInfo.InvariantCulture;
+            try
+            {
+                oDataAdapter.Fill(result);
+            }
+            catch (SQLiteException e)
+            {
+                this.ProcessException(e);
+            }
+
+            return result;
         }
 
-        public override void SqlStatement(string pSql, IDictionary<string, object> parameters)
+        public override void SqlStatement(string pSql, IDictionary<string, Object> parameters)
```

Comment:
the parameters dictionary is not being used here. please include those parameters in the sql statement.